charts/aws-ebs-csi-driver/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Bump driver version to `v1.30.0`
 * Update voluemessnapshotcontents/status RBAC ([#1991](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1991), [@AndrewSirenko](https://github.com/AndrewSirenko))
 * Upgrade dependencies ([#2016](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2016), [@torredil](https://github.com/torredil))
+* Add scrape interval ([#2034](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2034), [@omerap12][https://github.com/omerap12])
```

Comment:
Please drop this diff, this change is not in `v2.30.0` (it will be released in `v2.31.0` and the changelog for that will be generated when it is released).

Reply:
No problem